templates/Dockerfile.txt

```diff
@@ -1,11 +1,12 @@
-<% if(!coreclr){ %>FROM microsoft/aspnet:1.0.0-rc1-update1<% } %><% if(coreclr){ %>FROM microsoft/aspnet:1.0.0-rc1-update1-coreclr<% } %>
+<% if(!coreclr){ %>FROM microsoft/dotnet<% } %><% if(coreclr){ %>FROM microsoft/dotnet:core<% } %>
 
-RUN printf "deb http://ftp.us.debian.org/debian jessie main\n" >> /etc/apt/sources.list
-RUN apt-get -qq update && apt-get install -qqy sqlite3 libsqlite3-dev && rm -rf /var/lib/apt/lists/*
+#Set urls and port environments
+ENV ASPNETCORE_URLS="http://*:5000"
```

Comment:
Current templates are missing the configuration part with environment variables.
For RC2, I added :
`var config = new ConfigurationBuilder().AddEnvironmentVariables("ASPNETCORE_").Build();` and `.UseConfiguration(config)` in Program.cs, and `ENV ASPNETCORE_SERVER.URLS="http://*:5000"` in my Dockerfile and it works just fine! Looks like I'll have to use `ENV ASPNETCORE_URLS="http://*:5000"` post-RC2.